dll/win32/getuname/lang/ro-RO.rc

```diff
@@ -7983,6826 +7983,9449 @@ BEGIN
-    10147 "Three-D Bottom-Lighted Rightwards Arrowhead"
-    10148 "Black Rightwards Arrowhead"
-    10149 "Heavy Black Curved Downwards And Rightwards Arrow"
-    10150 "Heavy Black Curved Upwards And Rightwards Arrow"
-    10151 "Squat Black Rightwards Arrow"
-    10152 "Heavy Concave-Pointed Black Rightwards Arrow"
-    10153 "Right-Shaded White Rightwards Arrow"
-    10154 "Left-Shaded White Rightwards Arrow"
-    10155 "Back-Tilted Shadowed White Rightwards Arrow"
-    10156 "Front-Tilted Shadowed White Rightwards Arrow"
-    10157 "Heavy Lower Right-Shadowed White Rightwards Arrow"
-    10158 "Heavy Upper Right-Shadowed White Rightwards Arrow"
-    10159 "Notched Lower Right-Shadowed White Rightwards Arrow"
-    10160 "Curly Loop"
-    10161 "Notched Upper Right-Shadowed White Rightwards Arrow"
-    10162 "Circled Heavy White Rightwards Arrow"
-    10163 "White-Feathered Rightwards Arrow"
-    10164 "Black-Feathered South East Arrow"
-    10165 "Black-Feathered Rightwards Arrow"
-    10166 "Black-Feathered North East Arrow"
-    10167 "Heavy Black-Feathered South East Arrow"
-    10168 "Heavy Black-Feathered Rightwards Arrow"
-    10169 "Heavy Black-Feathered North East Arrow"
-    10170 "Teardrop-Barbed Rightwards Arrow"
-    10171 "Heavy Teardrop-Shanked Rightwards Arrow"
-    10172 "Wedge-Tailed Rightwards Arrow"
-    10173 "Heavy Wedge-Tailed Rightwards Arrow"
-    10174 "Open-Outlined Rightwards Arrow"
-    10175 "Double Curly Loop"
-    10176 "Three Dimensional Angle"
-    10177 "White Triangle Containing Small White Triangle"
+    9919 "Cheie pătrată"
+    9920 "Pion alb de damă"
+    9921 "Pion alb de rege"
+    9922 "Pion negru de damă"
+    9923 "Pion negru de rege"
+    9924 "Om de zăpadă fără zăpadă"
+    9925 "Soare în spatele norilor"
+    9926 "Ploaie"
+    9927 "Om de zăpadă negru"
+    9928 "Nor și ploaie cu tunete"
+    9929 "Piesă Shogi devenită albă"
+    9930 "Piesă Shogi devenită neagră"
+    9931 "Romb alb în pătrat"
+    9932 "Traversarea benzilor"
+    9933 "Mașină adaptată persoanelor cu handicap"
+    9934 "Ofiucus"
+    9935 "Târnăcop"
+    9936 "Mașină care alunecă"
+    9937 "Casca cu cruce alba"
+    9938 "Benzi de trecere încercuite"
+    9939 "Lanțuri"
+    9940 "Intrarea interzisă"
+    9941 "Trafic alternativ pe sens unic din stânga"
+    9942 "Trafic negru în două sensuri din stânga"
+    9943 "Trafic alb în două sensuri din stânga"
+    9944 "Îmbinarea benzii din stânga neagră"
+    9945 "Îmbinarea benzii din stânga albă"
+    9946 "Semnul de conducere lentă"
+    9947 "Triunghi alb îngroșat în jos"
+    9948 "Intrarea din stânga închisă"
+    9949 "Saltire pătrată"
+    9950 "Diagonală care se încadrează în cerc alb în pătrat negru"
+    9951 "Camion negru"
+    9952 "Intrare din stânga restricționată 1"
+    9953 "Intrare din stânga restricționată 2"
+    9954 "Simbolul astronomic pentru Uranus"
+    9955 "Cerc îngroșat cu contur și două puncte deasupra"
```

Comment:
I think you want to reconsider „contur” as translation for „stroke”. In many other instances you've used „tăiat” or „cu tăietură”.